plugins/scanners/scancode/src/main/kotlin/ScanCode.kt

```diff
@@ -90,7 +97,14 @@ class ScanCode internal constructor(
             }
         }
 
-    override val configuration by lazy { config.commandLine.joinToString(" ") }
+    override val configuration by lazy {
```

Comment:
Adding the per file findings as entries into the text location based license findings which correspond to a line range an a specific file, also implies that the existing license finding curation mechanism becomes applicable to these per file entries. 

So, from the perspective of creating the license finding curations, I wonder about the implication such as would we accept (e.g. in ort-config) also license finding curations which match such per file license findings?

Reply:
> would we accept (e.g. in ort-config) also license finding curations which match such per file license findings?

I think we should, mostly for two reasons:
- there's hardly any negative impact compared to not doing so,
- we should also accept curations for other supported scanners than ScanCode, and these other scanners *could* theoretically find exactly the file-level finding that ScanCode finds. If we'd accept such a curation for another scanner, we should accept it for ScanCode, too. (This somewhat reminds me of @mnonnenmacher's remark that our license finding curations should also include the scanner name and version.)